sdk/maps/Azure.Maps.Search/tests/SearchClientTestEnvironment.cs

```diff
@@ -15,6 +15,6 @@ public class SearchClientTestEnvironment : TestEnvironment
                 return endpoint != null ? new Uri(endpoint) : null;
             }
         }
-        public string MapAccountClientId => GetRecordedVariable("AZMAPS_CLIENT_ID");
+        public string MapAccountClientId => GetRecordedVariable("MAPS_CLIENT_ID");
```

Comment:
Should this match the name of the output comming from the test-resources.json file? https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/maps/test-resources.json#L44 Similar question for the ENDPOINT_URL above.

Reply:
The new value is correct, as it follows the `<service directory>_CLIENT_ID` convention for env vars that gets set by the deploy script, not the ARM template.